tests/integration/test_github.py

```diff
@@ -181,6 +181,11 @@ def test_pr_comments_updates(self):
 
 
 class GenericCommands(GithubTests):
+    def test_add_user(self):
+        project = self.service.get_project(repo="clynica", namespace="Beauth")
+        data = project.add_user("lachmanfrantisek", AccessLevel.pull)
+        assert data is None
```

Comment:
This check is misleading and unnecessary:
- We don't return any data from the method.
- It will fail with an exception if there is any problem. We don't need to check the return value.